src/lib/device/chunk_info.cpp

```diff
@@ -0,0 +1,36 @@
+/*********************************************************************************
```

Comment:
Is there a reason we didn't do this inside chunk_info.h itself and needed to define another .cpp file?